application/app/models/project.rb

```diff
@@ -2,6 +2,7 @@
 
 class Project < ApplicationDiskRecord
   include ActiveModel::Model
+  include YamlStorage
```

Comment:
Isn't it better to `include YamlStorage` in the parent class `ApplicationDiskRecord` instead and not in all children classes?

Reply:
yes - will update

Reply:
fixed